compose-network/mirror-node/application.yml

```diff
@@ -1,55 +0,0 @@
-hedera:
```

Comment:
This file should not be deleted. Doing so prevents users from using an out of the box run using docker compose.